markdown.html.markdown

```diff
@@ -153,12 +153,11 @@ inside your code -->
 John didn't even know what the `go_to()` function did!
 
 <!-- In Github Flavored Markdown, you can use a special syntax for code -->
-
-\`\`\`ruby <!-- except remove those backslashes when you do this, just ```ruby ! -->
-def foobar
-    puts "Hello world!"
-end
-\`\`\` <!-- here too, no backslashes, just ``` -->
+    ```ruby
```

Comment:
If the indenting is necessary, please make a note of that in the doc. It currently does not state *why* there is an indent.